models/topics.model.js

```diff
@@ -52,3 +41,26 @@ exports.getArticlesData = () => {
     return arrOfArticlesRows;
   });
 };
+
+exports.getCommentsDataByArticleId = (id) => {
+  return db
+    .query(
+      "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC",
+      [id]
+    )
+    .then(({ rows }) => {
+      return rows;
+    });
+};
+
+exports.addCommentToArticle = (id, { username, body }) => {
+  if (typeof username !== "string" || typeof body !== "string") {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
```

Comment:
If the type of your input is incorrect, PSQL will likely throw an error for you. No harm in throwing a promise manually though!